### ISSUE TITLE
Snap permission errors causing JavaScript error when launching app on Ubuntu/Linux

### ISSUE THREAD

**Issue by Clae224** [Apple-Music-Electron/Apple-Music-Electron#151]
After installing v2.4.0 with `sudo snap install apple-music-electron`, the app cannot launch as there are permission issues.

The initial error is:
```
Uncaught Exception:
Error: EPERM: operation not permitted, copyfile '/run/user/1000/snap.apple-music-electron/.org.chromium.Chromium.IJYQBF' -> '/home/clae/snap/apple-music-electron/common/.cache/Apple Music/Themes/README.md'
...
```

![Screenshot_20210728_194259](https://user-images.githubusercontent.com/17095335/127283791-6a9f4a94-ff0c-435f-9f24-13da5744ab67.png)

After some digging, I found this was likely caused by snap's permissions - according to [this page](https://manpages.ubuntu.com/manpages/hirsute/man8/snap-confine.8.html), it seems the EPERM error originates from a seccomp permission violation by the snap program. Taking this into account, I ran `sudo snap remove apple-music-electron --purge` and `sudo snap install apple-music-electron --devmode` to run the app without the usual sandboxing. This lets me run the app but I cannot log in due to the "Apple Music is not available in your region" message, trying Ctrl + Alt + S to open settings doesn't work so I cannot turn on authmode. Quitting and relaunching the app then fails, with a different permission error:

```
Uncaught Exception:
Error: EACCES: permission denied, lstat '/home/clae/snap/apple-music/electron/common/.cache/Apple Music/Themes/README.md'
...
```
![Screenshot_20210728_195701](https://user-images.githubusercontent.com/17095335/127286413-778ba0d4-db89-4e99-a171-eeebd2738b1e.png)

I cannot `chmod -R` the directory above since it's a snap which uses read-only files.

I also tried to work around this by installing the AppImage instead of the snap, only to be met with the same error but different directories: 

![Screenshot_20210728_200646](https://user-images.githubusercontent.com/17095335/127287251-11977c8c-7875-4673-8ec1-8f63e9b73891.png)

A workaround at this stage is to give more permissions to that folder above and the AppImage launches. But I still cannot log in due to the "not supported in your region" issue even when authmode is enabled. But that's a separate issue from the snap issue.

I am running Ubuntu 20.04.2 LTS x86_64 with KDE Plasma.

**Comment by infinytum** [comment]
This error also came up with the AppImage on Manjaro. Was able to fix it by opening up permissions on the folder too. Auth mode worked for me though.

**Comment by child-duckling** [comment]
This also happens on macOS

**Comment by coredev-uk** [comment]
Fixed in 00727a6
